### PR TITLE
[Docs] Updated adapter docs

### PR DESCRIPTION
**Pull request by chandrashritii**
**Description**

This PR will:
- Update the SMI conformance feature description in the Meshery adapter docs.
- Fix the broken link for the [Lifecycle Management](https://meshery.layer5.io/docs/functionality/lifecycle-management) page.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by akashgp09 — COMMENTED** [review]
Great 💯

File: docs/_adapters/istio/istio.md [hunk 1]
```diff
@@ -38,18 +38,9 @@ Once BookInfo is deployed, you can use Meshery to apply configuration to control
...
-Defining "Conformance" - It is important to acknowledge that conformance consists of both capabilities and compliance status. We define conformance as a combination of these two concepts.
-
-1. SMI Conformance acknowledges that
-   ...some participating service meshes may conscientiously never fully implement functions (SMI specs)...
-
-2. SMI Conformance identifies
-   ...a difference between full implementation of a specification and compliance with the portions that it implements...
-
-[Learn more](https://meshery.layer5.io/docs/functionality/smi-conformance) about SMI conformance capabilities here...
+Meshery allows you to analogize the compliance status and feature capabilities of your service mesh. This allows users to easily compare and contrast functional differences between service meshes and check whether their service mesh is conformant with the SMI specification.
```
Comment (leecalcote) [inline docs/_adapters/istio/istio.md:41]:
```suggestion
Meshery allows you to analyze the compliance status and functional capabilities of your service mesh. This allows you to compare high-level functional differences between service meshes and verify whether your service mesh is conformant with the SMI specification.
```

File: docs/_adapters/istio/istio.md [hunk 2]
```diff
@@ -38,18 +38,9 @@ Once BookInfo is deployed, you can use Meshery to apply configuration to control
...
 
+Learn more about the SMI specification and our conformance testing practices [here.](https://meshery.layer5.io/docs/functionality/smi-conformance)
```
Comment (leecalcote) [inline docs/_adapters/istio/istio.md:43]:
```suggestion
Learn more about the SMI specification and [Meshery's conformance test suite](https://meshery.layer5.io/docs/functionality/smi-conformance).
```

File: docs/_adapters/istio/istio.md (hunk 1 above)
Comment (leecalcote) [reply docs/_adapters/istio/istio.md:41]: @chandrashritii

File: docs/_adapters/istio/istio.md (hunk 2 above)
Comment (leecalcote) [reply docs/_adapters/istio/istio.md:43]: @chandrashritii these changes need propagated to the other adapters, too.

**Review by leecalcote — APPROVED** [review]
Looks great, @chandrashritii

**Review by kushthedude — COMMENTED** [review]
Please sign off commits from the next PR

**Pull request merged**